www/source/partials/docs/_glossary-services.html.md.erb

```diff
@@ -1,8 +1,4 @@
----
-title: Services
----
-
-# Services
+# <a name="glossary-services">Services</a>
 
 A service in Habitat is defined as a Habitat package running under a Habitat supervisor. Services are always run within named service groups, even if there is only one.
```

Comment:
Supervisor should be capitalized. 

Sometimes we're going to have multiple packages i.e. composite plans